### PR TITLE
[TDV-337] Fixed a bug with cannot invest if asset balance is 0 on Invest form

### PR DESCRIPTION
**Pull request by Electr1Xx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ for further information about branching and tagging conventions.
 - Invalid start date for sales that are already started
 - Fixed a bug with impossible order cancelling if user has insufficient balance
 - Fixed a bug with chart line rendering
+- Fixed a bug with cannot invest if asset balance is 0 on `Invest` form
```
Comment (letehaha) [inline CHANGELOG.md]: Don't add a `Fixed` at the sentencing front. You already have a `Fixed` title above. Also, this message describes nothing. That's was a bug. Which bug, how does component works right now?

I think you need to move PR describe under the `Changed` title and write something like:
`Warning/message about insufficient balance in the "Invest" form`

File: src/vue/forms/InvestForm.vue [hunk 2]
```diff
@@ -158,23 +158,6 @@
       </form>
     </template>
 
-    <template v-else-if="isLoaded && isAllowedAccountType">
-      <no-data-message
-        icon-name="alert-circle"
-        :title="'invest-form.insufficient-balance-title' | globalize"
-        :message="'invest-form.insufficient-balance-desc' | globalize"
```
Comment (letehaha) [inline src/vue/forms/InvestForm.vue]: You did not remove this `invest-form.insufficient-balance-title` `invest-form.insufficient-balance-desc` translations.

File: src/vue/forms/InvestForm.vue [hunk 3]
```diff
@@ -158,23 +158,6 @@
...
-      >
-        <router-link
-          :to="vueRoutes.movements"
-          tag="button"
-          type="button"
-          class="app__button-raised invest-form__discover-tokens-btn"
```
Comment (letehaha) [inline src/vue/forms/InvestForm.vue]: Now we have useless `invest-form__discover-tokens-btn` in the CSS. When you deleted some stuff, make sure that you deleted it from everywhere

File: src/vue/forms/InvestForm.vue [hunk 4]
```diff
@@ -158,23 +158,6 @@
...
-        >
-          {{ 'invest-form.deposit-btn' | globalize }}
```
Comment (letehaha) [inline src/vue/forms/InvestForm.vue]: Useless `invest-form.deposit-btn` in the `en.json` and `reit-en.json` files

File: src/vue/forms/InvestForm.vue (hunk 2 above)
Comment (Electr1Xx) [reply src/vue/forms/InvestForm.vue]: It's not in `reit-en.json`

File: src/vue/forms/InvestForm.vue (hunk 4 above)
Comment (Electr1Xx) [reply src/vue/forms/InvestForm.vue]: It's not in `reit-en.json`

**Pull request merged**